js/app.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+console.log('js is connected.');
+
```

Comment:
Global variables for displaying the images will then look like this:

let image1 = document.querySelector('section img:first-child');
let image2 = document.querySelector('section img:nth-child(2)');
let image3 = document.querySelector('section img:nth-child(3)');